contracts/modules/credit/SpigotedLine.sol

```diff
@@ -229,9 +233,10 @@ contract SpigotedLine is ISpigotedLine, LineOfCredit, ReentrancyGuard {
     }
 
     /**
-     * @notice - allow borrower to call functions on their protocol to maintain it and keep earning revenue
+     * @notice - Sets or resets the whitelisted functions that a Borrower [Operator] is allowed to perform on the revenue generating contracts 
+                whilst the Spigot is attached.  Triggers updateWhitelistedFunction() in Spigot.sol
```

Comment:
I like referencing the side effects in the Spigot. Instead of documenting the functoin call which we can already see in the code it would be great if you mention the event emitted (UpdateWhitlistedFunction) or the internal state changes (spigot.whitelistedFunctions)